yarn-project/aztec.js/src/contract/deploy_method.ts

```diff
@@ -114,6 +115,19 @@ export class DeployMethod<TContract extends ContractBase = Contract> extends Bas
     return mergeExecutionPayloads(exec);
   }
 
+  /**
+   * Simulate a deployment and profile the gate count for each function in the transaction.
+   * @param options - Same options as `simulate`, plus profiling method
```

Comment:
There is no `simulate` fn in this class nor `BaseContractInteraction`

Reply:
Fixed